conf.d/git.fish

```diff
@@ -0,0 +1,18 @@
+set -l event_name_base (basename (status -f) .fish)
```

Comment:
Simply use `git`? I know I used to recommend this before, but I find this damages readability now.

Reply:
Sure! Will update. btw haven't heard back from @jhillyerd on the original issue as to how he wants to proceed with supporting Fisher/OMF (via what's in this PR vs separate branches).